draft-ietf-httbis-h3-websockets.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Bootstrapping WebSockets with HTTP/3"
+docname: draft-ietf-httbis-h3-websockets.md
+category: info
+
+ipr: trust200902
+area: TODO
```

Comment:
```suggestion
area: ART
```

Reply:
Thanks! I applied the change to the renamed file.